services/spiget/spiget-versions.service.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const { BaseSpigetService, documentation } = require('./spiget-base')
+
+module.exports = class SpigetTestedVersions extends BaseSpigetService {
+  static get route() {
+    return {
+      base: 'spiget/versions',
```

Comment:
Let's update the route (and file name) to both be consistent and a bit more explicit that this is for `tested-versions`